name.abuchen.portfolio.tests/src/name/abuchen/portfolio/util/TradeCalendarTest.java

```diff
@@ -159,66 +158,97 @@ public void testMovingHolidays()
     }
 
     @Test
-    public void testNYSE()
+    public void testTradeCalenderNYSE()
     {
         // See https://www.nyse.com/markets/hours-calendars
 
         TradeCalendar calendar = TradeCalendarManager.getInstance("nyse");
 
-        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-10")), is(false)); // regular Monday
-        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-11")), is(true)); // closed after terrorist attacks (9/11)
-        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-14")), is(true)); // ... still closed
-        assertThat(calendar.isHoliday(LocalDate.parse("2001-09-17")), is(false)); // reopened the following Monday
-
-        assertThat(calendar.isHoliday(LocalDate.parse("2020-01-01")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2020-01-20")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2020-02-17")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2020-04-10")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2020-05-25")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2020-07-03")), is(true));
-        assertThat(calendar.isHoliday(LocalDate.parse("2020-09-07")), is(true));
```

Comment:
Removed despite claiming that no tests were removed.

Reply:
https://github.com/buchen/portfolio/blob/718f13d64d6f4d6526de484c948a4c4064f393b1/name.abuchen.portfolio.tests/src/name/abuchen/portfolio/util/TradeCalendarTest.java#L227
Here... and is not removed